docs/Makefile.am

```diff
@@ -74,6 +74,7 @@ EXTRA_DIST =                                    \
  KNOWN_BUGS                                     \
  MAIL-ETIQUETTE                                 \
  MQTT.md                                        \
+ NEW-PROTOCOL.md                                \
```

Comment:
Curiosity killed the cat, why do we use all-caps for (most) documentation filenames? Legacy?

Reply:
Plain legacy and old habits... maybe I should rather break that off right now and do lower case here!